src/App.js

```diff
@@ -1,104 +1,99 @@
-import './App.css';
-import Todo from "./components/Todo"
-import Form from "./components/Form"
-import FilterButton from './components/FilterButton';
+import "./App.css";
+import Todo from "./components/Todo";
+import Form from "./components/Form";
+import FilterButton from "./components/FilterButton";
 
 import React, { useEffect, useState } from "react";
 import { nanoid } from "nanoid";
 
 function App(props) {
-  
   const [tasks, setTasks] = useState(props.tasks);
   const [filter, setFilter] = useState("All");
-  
-  
-  function addTask(name) {
-    if (name==="") return;
+
+  const addTask = (name) => {
+    if (name === "") return;
     let newTask = {
-      id: "task-"+nanoid(),
+      id: "task-" + nanoid(),
       name: name,
-      completed: false
+      completed: false,
     };
     setTasks([...tasks, newTask]);
```

Comment:
looking at this code, i think you should use `const` here rather than `let` because no where in that code do you reference it again.